src/libcore/iter/traits/accum.rs

```diff
@@ -9,9 +9,9 @@ use crate::ops::{Add, Mul};
 /// [`FromIterator`] this trait should rarely be called directly and instead
 /// interacted with through [`Iterator::sum`].
 ///
-/// [`sum`]: ../../std/iter/trait.Sum.html#tymethod.sum
-/// [`FromIterator`]: ../../std/iter/trait.FromIterator.html
-/// [`Iterator::sum`]: ../../std/iter/trait.Iterator.html#method.sum
+/// [`sum`]: #tymethod.sum
```

Comment:
Does this not have an equivalent?

Reply:
This fragment link is fine as the sum method is belong to Sum trait.

Reply:
This is https://github.com/rust-lang/rust/issues/51284, which is a project all on its own. If @lzutao checked that it works I'm fine with it.

Reply:
Can this not be `Sum::sum` or `Self::sum`?

Reply:
Is it documented in Intra RFC?